introduction.md

```diff
@@ -1,2 +1,6 @@
 With several years of experience in the Software Development Industry, Cristián is an innovative Web Developer. He has been working in areas such as testing, development, presentation layer development, creation of new software products and also researching and providing services as an independent consultant of new technologies. He has implemented business solutions for companies in the region such as CMS, CRM and LMS and has dictated OOP courses and basic training on IA. He developed an automation robot for GUI testing, a management system and designed sites for companies in Argentina. In the last four years worked on projects for global companies such us U.S. Cellular, Intel, Microsoft and MTV Networks among others...
 My favourite path in CodeSchool is: JavaScript
+
+## Deadly Skills
+* Javascript
+* Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?